workshop/mathIsFun/ComplexNumber.cpp

```diff
@@ -0,0 +1,165 @@
+#include "ComplexNumber.h"
+#include <string>
+
+/*
+ * default constructor, initialize number to be zero
```

Comment:
I don't think you need to document each function twice (both in the header and in the source file) it might even be worse than documenting once - since when someone else will change your code he/she might only fix one docstring accidently, which will mean there will be 2 different docstrings for the same function in the project...